test/tuner_test.py

```diff
@@ -77,7 +77,7 @@ def run(dispatch_type):
     for dispatcher_name in dipsatcher_list:
         try:
             # sleep 5 seconds here, to make sure previous stopped exp has enough time to exit to avoid port conflict
-            time.sleep(5)
+            time.sleep(6)
```

Comment:
why change this? shall we change the comments at the same time? "# sleep 5 seconds here, to make sure previous stopped exp has enough time to exit to avoid port conflict"....

Reply:
Thanks, updated the comments.
The reason to change this is:  after this tuner/assessor queue change, if there is no commands in queues, it will still possible to wait up to 3 seconds after dispatcher receives TERMINATE command. So it may take a little bit longer time to end for a normal exepriment.